documents/attribute.go

```diff
@@ -96,14 +118,11 @@ type AttrVal struct {
 	Str       string
 	Bytes     []byte
 	Timestamp *timestamp.Timestamp
+	Signed    Signed
```

Comment:
Having a new type, implies that we only support []byte values and we will not need to perform any specific/typed operation over values of type `signed`?

Reply:
not sure I understand this